Compilation failure

```
% git checkout 8.3.0
% cargo build --no-default-features --features build,git,rustc
   Compiling thiserror v1.0.56
   Compiling darling_macro v0.14.4
   Compiling vergen v8.3.0 (/home/nicolas/git/vergen/vergen)
   Compiling gix-hash v0.14.1
   Compiling gix-path v0.10.3
   Compiling gix-date v0.8.3
   Compiling gix-validate v0.8.3
   Compiling gix-chunk v0.4.7
   Compiling gix-quote v0.4.10
   Compiling gix-bitmap v0.2.10
   Compiling gix-packetline-blocking v0.17.2
   Compiling gix-packetline v0.17.2
error[E0433]: failed to resolve: use of undeclared crate or module `gix`
  --> vergen/src/feature/git/gix.rs:21:5
   |
21 | use gix::{head::Kind, Commit, Head, Id, Repository};
   |     ^^^ use of undeclared crate or module `gix`
(...)
```

--- a/README.md
+++ b/README.md
@@ -57,7 +57,7 @@ vergen = { version = "8.2.10", features = ["git","gitcl","gitoxide"] }
 and prior to **8.3.0** `vergen` will not compile with both `gitcl` and `gitoxide` as features.
 
 As a workaround, you can use `cargo tree -f "{p} {f}" | grep vergen` to determine the feature list cargo has set for `vergen`.  If
-a `git` backend has already been determined you will be able to use that without declaring those features in your dependency list.
+a `git` backend has already been determined you will be able to use that without declaring those features in your dependency list.  This is not perfect as this leaves you at the mercy of your dependency and the git feature they selected, but it's a workaround until version 9 comes out.
 
 #### fancy-lib `Cargo.toml`
 ```toml
@@ -78,7 +78,7 @@ vergen = "8.2.10"
 vergen = { version = "8.2.10", features = ["git","gitcl"] }
 ```
 ## `Cargo` feature unification for `vergen` versions 8.3.0 and beyond
-`vergen` will accept `gitcl`,`git2`, and `gitoxide` as features.  If more than one of them is included, `vergen` will select `gitcl` before `git2` and `git2` before `gitoxide`.   If none are selected (for example, if you enable `git`, but don't select a backend feature), `gitoxide` will be selected.
+`vergen` will accept `gitcl`,`git2`, and `gitoxide` as features.  If more than one of them is included, `vergen` will select `gitcl` before `git2` and `git2` before `gitoxide`.
 
 ## Notes about the optional `git2` dependency
 `git2` picked up some [security related features](https://github.blog/2022-04-12-git-security-vulnerability-announced/).  In docker environments especially, this requires a `safe.directory` configuration.   There are a couple methods for achieving this.

--- a/vergen/src/lib.rs
+++ b/vergen/src/lib.rs
@@ -185,7 +185,7 @@
 //! and prior to **8.3.0** `vergen` will not compile with both `gitcl` and `gitoxide` as features.
 //!
 //! As a workaround, you can use `cargo tree -f "{p} {f}" | grep vergen` to determine the feature list cargo has set for `vergen`.  If
-//! a `git` backend has already been determined you will be able to use that without declaring those features in your dependency list. This is not perfect 
+//! a `git` backend has already been determined you will be able to use that without declaring those features in your dependency list. This is not perfect
 //! as this leaves you at the mercy of your dependency and the git feature they selected, but it's a workaround until version 9 comes out.
 //!
 //! #### fancy-lib `Cargo.toml`

--- a/vergen/src/lib.rs
+++ b/vergen/src/lib.rs
@@ -185,7 +185,8 @@
 //! and prior to **8.3.0** `vergen` will not compile with both `gitcl` and `gitoxide` as features.
 //!
 //! As a workaround, you can use `cargo tree -f "{p} {f}" | grep vergen` to determine the feature list cargo has set for `vergen`.  If
-//! a `git` backend has already been determined you will be able to use that without declaring those features in your dependency list.
+//! a `git` backend has already been determined you will be able to use that without declaring those features in your dependency list. This is not perfect 
+//! as this leaves you at the mercy of your dependency and the git feature they selected, but it's a workaround until version 9 comes out.
 //!
 //! #### fancy-lib `Cargo.toml`
 //! ```toml
@@ -206,7 +207,7 @@
 //! vergen = { version = "8.2.10", features = ["git","gitcl"] }
 //! ```
 //! ### `Cargo` feature unification for `vergen` versions 8.3.0 and beyond
-//! `vergen` will accept `gitcl`,`git2`, and `gitoxide` as features.  If more than one of them is included, `vergen` will select `gitcl` before `git2` and `git2` before `gitoxide`.   If none are selected (for example, if you enable `git`, but don't select a backend feature), `gitoxide` will be selected.
+//! `vergen` will accept `gitcl`,`git2`, and `gitoxide` as features.  If more than one of them is included, `vergen` will select `gitcl` before `git2` and `git2` before `gitoxide`.
 //!
 //! ## Environment Variables
 //! `vergen` currently recognizes the following environment variables

